products/pixel.md

```diff
@@ -44,6 +53,7 @@ releases:
     eol: 2028-05-07
     discontinued: false
     link: https://en.wikipedia.org/wiki/Pixel_7a
+    supportedAndroidVersions: 13
```

Comment:
```suggestion
    supportedAndroidVersions: 13+
```

On https://en.wikipedia.org/wiki/Google_Pixel#Phones, isn't empty "Final OS" means all versions from "With OS" to the latest Android version ?

Reply:
Yes, that would be nice.

Reply:
Strange, on https://www.androidauthority.com/google-pixel-7a-review-3320910/ they say 7a received android 14. What do you think of this @paulverbeke ? Is there Google documentation that could confirm this ?

Reply:
I don't see any other evidence of this on the web. I don't think there's better documentation than the official Google pixel OS download page

Reply:
We decided to base this column on what can be found on gsmarena. Resolved.